src/overmap_ui.cpp

```diff
@@ -803,6 +803,7 @@ tripoint display( const tripoint &orig, const draw_data_t &data = draw_data_t()
                 color_notes += string_format( "%s:%s, ", color_pair.first.c_str(),
                                               _( color_pair.second.c_str() ) );
             }
+            color_notes = color_notes.replace(color_notes.end() - 2, color_notes.end(), ".");
```

Comment:
Should be astyled (single whitespace after `(` and before `)` ):

```CPP
color_notes = color_notes.replace( color_notes.end() - 2, color_notes.end(), "." );
```

Reply:
Thanks! I noticed the failed build's output but I didn't know where to check for the style guidelines. I'm gonna fix it right away.